docs/content/ui/components/badges.md

```diff
@@ -99,14 +131,55 @@ Place the badge within an `<a>` element if you want it to perform the function o
 
 ## Button with badge
 
-Badges can be used as part of links or buttons to provide, for example, a counter.
+Badges can be used as part of links or buttons to provide, for example, a counter. Use the `.badge-notification` class to create a notification badge. This class will add a notification positioning to the badge, so it will be placed in the top right corner of the button.
+
+If you dont provide a text to the badge, it will be hidden. This is useful for creating a notification button with a red dot.
 
 {% capture html -%}
 <button type="button" class="btn">
-	Notifications <span class="badge bg-red-lt ms-2">4</span>
+	Notifications {% include "ui/badge.html" text="2" color="red" class="ms-2" -%}
 </button>
 <button type="button" class="btn">
-  Notifications <span class="badge bg-green-lt ms-2">4</span>
+  Inbox {% include "ui/badge.html" text="4" color="red" class="badge-notification" -%}
+</button>
+<button type="button" class="btn">
+  Profile {% include "ui/badge.html" text="" color="red" class="badge-notification" -%}
 </button>
 {%- endcapture %}
 {% include "docs/example.html" html=html centered %}
+
+## Animated badges
+
+You can use the `.badge-blink` class to create a blinking effect. This class will add a CSS animation to the badge, so it will blink to draw attention.
+
+{% capture html -%}
+<button type="button" class="btn">
+  Profile {% include "ui/badge.html" text="" color="red" class="badge-notification badge-blink" -%}
+</button>
+{% endcapture %}
+{% include "docs/example.html" html=html centered %}
+
+## Size Options
+
+Use `.badge-sm` or `.badge-lg` to change Badge size according to your needs. The default size is `.badge` and it is used in the examples above.
+
+{% capture html -%}
+<div>
+  {% include "ui/badge.html" color="primary" scale="sm" text="New" class="badge-sm" -%}
+  {% include "ui/badge.html" color="primary" scale="sm" text="1" class="badge-pill" -%}
+</div>
+<div>
+  {% include "ui/badge.html" color="primary" text="New" class="badge-sm" -%}
+  {% include "ui/badge.html" color="primary" text="1" class="badge-pill" -%}
+</div>
+<div>
+  {% include "ui/badge.html" color="primary" scale="lg" text="New" class="badge-sm" -%}
+  {% include "ui/badge.html" color="primary" scale="lg" text="1" class="badge-pill" -%}
+</div>
+{%- endcapture %}
+{% include "docs/example.html" html=html centered vertical %}
+
+
+## More examples 
+
+If you want to see more examples of badges, you can check out the [Bootstrap documentation](https://getbootstrap.com/docs/5.3/components/badge/) for badges. You can also find more examples in the Tabler [Badges](https://preview.tabler.io/badges.html) preview.
```

Comment:
Not necessarily in this PR, but as I mention in #2339, would it be feasible to reduce the use of hardcoded links to the Bootstrap documentation by making use of the previously added `bootstrapLink` variables?

Similarly, is it worth using `{{ site.previewUrl }}` in the badges link here?

Reply:
I'll fix it in another PR